lib/v2/36kr/hot-list.js

```diff
@@ -0,0 +1,86 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const CryptoJS = require('crypto-js');
+
+const categories = {
+    24: {
+        title: '24小时热榜',
+        regex: 'hotlist":{"code":0,"data',
+    },
+    renqi: {
+        title: '资讯人气榜',
+        regex: 'topList',
+    },
+    zonghe: {
+        title: '资讯综合榜',
+        regex: 'hotList',
+    },
+    shoucang: {
+        title: '资讯综合榜',
+        regex: 'collectList',
+    },
+};
+
+module.exports = async (ctx) => {
+    const category = ctx.params.category ?? '24';
+
+    const rootUrl = 'https://www.36kr.com';
+    const currentUrl = category === '24' ? rootUrl : `${rootUrl}/hot-list/catalog`;
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+
+    const data = JSON.parse(response.data.match(new RegExp('"' + categories[category].regex + '":(\\[.*?\\])'))[1]);
+
+    let items = data
+        .slice(0, ctx.query.limit ? parseInt(ctx.query.limit) : 30)
+        .filter((item) => item.itemType !== 0)
+        .map((item) => {
+            item = item.templateMaterial ?? item;
+            return {
+                title: item.widgetTitle.replace(/<\/?em>/g, ''),
+                author: item.author,
+                pubDate: parseDate(item.publishTime),
+                link: `${rootUrl}/p/${item.itemId}`,
+                description: item.widgetContent ?? item.content,
```

Comment:
```suggestion
                description: item.summary,
```
I can't find `item.author`, `item.widgetContent` or `item.content`.